src/System.Runtime.Serialization.Xml/src/System.Runtime.Serialization.Xml.csproj

```diff
@@ -14,20 +14,31 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50_Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
 
   <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50aot'">
     <ProjectJson>netcore50aot\project.json</ProjectJson>
     <ProjectLockJson>netcore50aot\project.lock.json</ProjectLockJson>
   </PropertyGroup>
-
+  
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcore50aot'" >
+    <TargetingPackReference Include="System.Private.CoreLib" />
+    <TargetingPackReference Include="System.Private.DataContractSerialization" />
+  </ItemGroup>
   <ItemGroup>
```

Comment:
It was kind of hard to figure out which references was part of which configuration. Could you try to move the conditions to the ItemGroup and group them for each configuration to help with that.